Sources/CocoaLumberjack/DDFileLogger.m

```diff
@@ -1228,14 +1228,18 @@ - (void)flush {
 
 - (void)lt_flush {
     NSAssert([self isOnInternalLoggerQueue], @"flush should only be executed on internal queue.");
-    if (@available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)) {
-        __autoreleasing NSError *error;
-        BOOL succeed = [_currentLogFileHandle synchronizeAndReturnError:&error];
-        if (!succeed) {
-            NSLogError(@"DDFileLogger: Failed to synchronize file: %@", error);
+    @try {
```

Comment:
Can we put the `@try` into the `else` branch? It's only necessary if the error-returning API is not available.

```objc
if (@available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)) {
    __autoreleasing NSError *error;
    BOOL succeed = [_currentLogFileHandle synchronizeAndReturnError:&error];
    if (!succeed) {
        NSLogError(@"DDFileLogger: Failed to synchronize file: %@", error);
    }
} else {
    @try {
        [_currentLogFileHandle synchronizeFile];
    } @catch (NSException *exception) {
        NSLogError(@"DDFileLogger: Failed to synchronize file: %@", exception);
    }
}
```

Reply:
thanks for review！

Yes，you are right

Reply:
done